setup.py

```diff
@@ -120,15 +120,15 @@ def read(*names, **kwargs):
     else:
         bash_file = "/dev/null"
 
-    argcomplete_command = 'eval "$(register-python-argcomplete retriever)"'
+    argcomplete_command = "eval $(register-python-argcomplete retriever)"
     with open(os.path.expanduser(bash_file), "a+") as bashrc:
         bashrc.seek(0)
         # register retriever for arg-completion if not already registered
         # whenever a new shell is spawned
         if argcomplete_command not in bashrc.read():
             bashrc.write(argcomplete_command + "\n")
             bashrc.close()
-    os.system("activate-global-python-argcomplete")
+    os.system("activate-global-python-argcomplete --user")
```

Comment:
is --user needed ?

Reply:
Yes, now having that wil lead to :
```bash

Installing bash completion script /etc/bash_completion.d/python-argcomplete
usage: activate-global-python-argcomplete [-h] [--dest DEST] [--user]
                                          [--no-defaults]
                                          [--complete-arguments ...]
activate-global-python-argcomplete: error: [Errno 13] Permission denied: '/etc/bash_completion.d/python-argcomplete'
Please try --user to install into a user directory, or --dest to specify the bash completion modules directory

```

Reply:
What OS are you using?

Reply:
linux

Reply:
I will test it with other OS, any one  else is welcome to help with the testing and report back